_includes/manuals/3.10/1.2_release_notes.md

```diff
@@ -0,0 +1,29 @@
+## Release notes for {{page.version}}
+
+This section will be updated prior to the next release.
+
+### Headline features
+
+### Upgrade warnings
+
+#### `ct_command` and `fcct_command` settings replaced with `(fc)ct_location` and `(fc)ct_arguments`
```

Comment:
I think we forgot to clear the nightly manual when branching 3.9. Would you mind doing so?